lib/swaps/SwapClient.ts

```diff
@@ -17,9 +17,14 @@ type ChannelBalance = {
   pendingOpenBalance: number,
 };
 
+export interface ConnectionVerified {
```

Comment:
We've been using `type` instead of `interface`, I think they are effectively the same but better to be consistent. Also `ConnectionVerified` is a bit confusing of a name for a type to me, how about `ConnectionInfo` or `SwapClientInfo`?